src/state/actions/mentee/index.js

```diff
@@ -8,9 +9,10 @@ export const MENTEE_ADD_SUCCESS = 'MENTEE_ADD_SUCCESS';
 
 export const postNewMenteeAccount = newAccount => {
   return async dispatch => {
+
     try {
       dispatch(setFetchStart());
-      const api = await axiosWithAuth().post(
+      const api = await axios().post(
         `${API_URL}application/new/mentee`,
```

Comment:
Good eye fixing this call while you were working on auth - a user does not need to be signed in in order to submit an application to become a user, so this is a sensible change.